src/Tizen.WebView/Tizen.WebView/Settings.cs

```diff
@@ -98,5 +116,113 @@ public int DefaultFontSize
+
+        /// <summary>
+        /// Whether the scripts can open windows.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool ScriptsCanOpenWindows
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_set(_handle, value);
+            }
+        }
+ 
+        /// <summary>
+        /// Enable or Disable force zoom.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool ForceZoom
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_force_zoom_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_force_zoom_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable or Disable text autosizing.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool TextAutosizing
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_text_autosizing_enabled_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_text_autosizing_enabled_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable or Disable text zoom.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool TextZoom
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_text_zoom_enabled_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_text_zoom_enabled_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable or Disable the usage of keypad without user action.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool UseKeypadWithoutUserAction
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_uses_keypad_without_user_action_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_uses_keypad_without_user_action_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Enable or Disable Extra feature such as "edge,enable", "zoom,enable", "longpress,enable"
+        /// "doubletap,enable" and "selection,magnifier".
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool ExtraFeature
```

Comment:
|ewk_settings_extra_feature_set| can set several extra features, because it is not fit for property.
Property has only one value at a time. What about implementing it as methods?

Reply:
OK